taichi/python/export_lang.cpp

```diff
@@ -463,7 +463,9 @@ void export_lang(py::module &m) {
           },
           py::arg("dt"), py::arg("shape"),
           py::arg("layout") = ExternalArrayLayout::kNull,
-          py::arg("zero_fill") = false, py::return_value_policy::reference)
+          py::arg("zero_fill") = false, py::return_value_policy::reference,
+          py::return_value_policy::reference)
```

Comment:
`py::return_value_policy::reference` repeats twice, what does this mean?

Reply:
oops it's a typo I think...